theme/snippets/news-feed.liquid

```diff
@@ -9,6 +11,6 @@
   {% endfor %}
   </div>
   {% if limit %}
-    <a class="uppercase serif-xl pt2 md:pt5 pb3 hover-lighten-black" href="/pages/news" aria-label="{{ 'snippets.news_feed.see_more_link.aria_label' | t }}">{{ 'snippets.news_feed.see_more_link.text' | t }}</a>
+    <a class="NewsFeed__button button--style-rounded-border-xs mt1_25" href="/pages/news" aria-label="{{ 'snippets.news_feed.see_more_link.aria_label' | t }}">{{ see_more_link_label }}</a>
```

Comment:
The "See more" link after the Homepage News Feed

(Thought I would make it editable since they have asked to edit text)